Kombit.Samples.CH.WebsiteDemo/Web.config

```diff
@@ -65,105 +67,138 @@
       </md:ContactPerson>
     </ServiceProvider>
     <RequestedAttributes>
-      <att name="dk:gov:saml:attribute:AssuranceLevel" isRequired="true" />
-      <att name="dk:gov:saml:attribute:SpecVer" isRequired="true" />
-      <att name="dk:gov:saml:attribute:KombitSpecVer" isRequired="true" />
-      <att name="dk:gov:saml:attribute:CvrNumberIdentifier" isRequired="false" />
-      <att name="dk:gov:saml:attribute:Privileges_intermediate" isRequired="false" />
+      <att name="dk:gov:saml:attribute:AssuranceLevel" isRequired="true"/>
+      <att name="dk:gov:saml:attribute:SpecVer" isRequired="true"/>
+      <att name="dk:gov:saml:attribute:KombitSpecVer" isRequired="true"/>
+      <att name="dk:gov:saml:attribute:Privileges_intermediate" isRequired="true"/>
+      <att name="dk:gov:saml:attribute:CvrNumberIdentifier" isRequired="false"/>
+      
+      <att name="https://data.gov.dk/model/core/specVersion" isRequired="true"/>
+      <att name="https://data.gov.dk/model/core/eid/privilegesIntermediate" isRequired="true"/>
+      <att name="https://data.gov.dk/concept/core/nsis/loa" isRequired="true"/>
+
+      <att name="https://data.gov.dk/model/core/eid/professional/cvr" isRequired="true"/>
+      <att name="https://data.gov.dk/model/core/eid/professional/orgName" isRequired="true"/>
+      
+      <att name="https://data.gov.dk/model/core/eid/bootstrapToken" isRequired="false"/>
+      <att name="https://data.gov.dk/concept/core/nsis/ial" isRequired="false"/>
+      <att name="https://data.gov.dk/concept/core/nsis/aal" isRequired="false"/>
+      <att name="https://data.gov.dk/model/core/eid/fulName" isRequired="false"/>
+      <att name="https://data.gov.dk/model/core/eid/firstName" isRequired="false"/>
+      <att name="https://data.gov.dk/model/core/eid/lastName" isRequired="false"/>
+      <att name="https://data.gov.dk/model/core/eid/email" isRequired="false"/>
+      <att name="https://data.gov.dk/model/core/eid/cprNumber" isRequired="false"/>
```

Comment:
Same comments as those in the /deployment folder

Reply:
fixed